documentation/create_docs.sh

```diff
@@ -7,7 +7,7 @@ echo "Removing build directory"
 rm -rf build
 
 echo "Creating docs"
-xsltproc --xinclude xsl/rhea_php.xsl rhea_doc.xml
+xsltproc --xinclude xsl/rhea_33_php.xsl rhea_33_doc.xml
```

Comment:
The name for 3.3 is `proteus`.  
Why using `rhea` ?

Reply:
phpBB 3.3 is still Rhea. phpBB 4.0 will be named Proteus.

Reply:
Hmmm… Okay.  
This has changed. Initially Proteus was for 3.3, but I don't follow discussion about the phpBB naming.
Thanks for the information